common/constants.go

```diff
@@ -0,0 +1,22 @@
+package common
+
+const (
+	// InstanceID instance ID key for logging details
+	InstanceID = "instanceID"
```

Comment:
I think I have been consistent. The constant starts with upper case as its public, but all parameters to functions or vars are instanceID